control-plane/agents/src/bin/core/volume/registry.rs

```diff
@@ -103,7 +103,7 @@ impl Registry {
     }
 
     /// Get all volumes
-    pub(super) async fn get_volumes(&self) -> Vec<Volume> {
+    pub(crate) async fn get_volumes(&self) -> Vec<Volume> {
```

Comment:
Since you're here, could you add missing `.` above? thanks

Reply:
fixed (plus some others)